### PR TITLE
[styles] Add more vending_machine types

### PR DESCRIPTION
**Pull request by dvdmrtnz**
Added `vending=coffee` and `vending=sweets` types to `amenity=vending_machine`.

### CODE REVIEW

File: data/mapcss-mapping.csv [hunk 1]
```diff
@@ -213,8 +213,8 @@ railway|rail|tunnel;[railway=rail][tunnel?];;name;int_name;212;
 man_made|wastewater_plant;213;
 man_made|flagpole;214;
 place|county;215;
-deprecated|deprecated;216;x
-deprecated|deprecated;217;x
+amenity|vending_machine|coffee;[amenity=vending_machine][vending=coffee];;name;int_name;216;
+amenity|vending_machine|sweets;[amenity=vending_machine][vending=sweets];;name;int_name;217;
```
Comment (vng) [inline data/mapcss-mapping.csv:217]: Less value (216, 217) means more frequent tag in OSM. Please, put this types somewhere around existing amenity=vending_machine.

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -3384,6 +3384,11 @@
     zh-Hans = 香烟售货机
     zh-Hant = 香菸自動販賣機
 
+  [type.amenity.vending_machine.coffee]
+    en = Coffee Dispenser
+    es = Máquina expendedora de café
+    fr = Distributeur de café
```
Comment (pastk) [inline data/strings/types_strings.txt:3390]: be = Кававы аўтамат
ru = Кофейный автомат
uk = Кавовий автомат

File: data/mapcss-mapping.csv (hunk 1 above)
Comment (vng) [reply data/mapcss-mapping.csv:217]: Here will be ok:
deprecated|deprecated;1061;x
deprecated|deprecated;1062;x

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -3485,6 +3490,11 @@
     zh-Hans = 公共交通售票机
     zh-Hant = 公共交通售票機
 
+  [type.amenity.vending_machine.sweets]
+    en = Sweets Dispenser
+    es = Máquina expendedora de dulces
+    fr = Distributeur de bonbons
```
Comment (pastk) [inline data/strings/types_strings.txt:3496]: be = Аўтамат з прысмакамі
ru = Автомат со сладостями
uk = Автомат із солодощами

File: data/categories.txt [hunk 4]
```diff
@@ -7966,6 +7966,11 @@ el:Αυτόματος πωλητής τσιγάρων
 sk:Automat na cigarety
 fa:ماشین سیگار فروشی
 
+amenity-vending_machine-coffee
+en:Coffee Dispenser
+es:Máquina expendedora de café
+fr:Distributeur de café
+
```
Comment (pastk) [inline data/categories.txt:7973]: en:4Coffee Dispenser|Coffee
be:3Кававы аўтамат|кава
ru:3Кофейный автомат|кофе
uk:3Кавовий автомат|кава

File: data/categories.txt [hunk 5]
```diff
@@ -8069,6 +8074,11 @@ vi:Máy bán vé phương tiện công cộng tự động
 zh-Hans:公共交通售票机
 zh-Hant:公共交通售票機
 
+amenity-vending_machine-sweets
+en:Sweets Dispenser
+es:Máquina expendedora de dulces
+fr:Distributeur de bonbons
```
Comment (pastk) [inline data/categories.txt:8080]: en:4Sweets Dispenser|Sweets|4Candies|Lollies
be:Аўтамат з прысмакамі|4прысмакі|4цукеркі
ru:Автомат со сладостями|4сладости|4конфеты
uk:Автомат із солодощами|4солодощі|4цукерки

**Review by pastk — COMMENTED** [review]
Great, thanks!

Other major use vending machines we don't have yet are
`food` ~2400
`newspapers` ~2900
`condoms` ~2300

File: data/mapcss-mapping.csv (hunk 1 above)
Comment (pastk) [reply data/mapcss-mapping.csv:217]: > Less value (216, 217) means more frequent tag in OSM. Please, put this types somewhere around existing amenity=vending_machine.

Wow, I didn't know that either, thanks!
Does it have any consequences e.g. performance- or space-wise?

**Comment by dvdmrtnz** [comment]
> Great, thanks!
> 
> Other major use vending machines we don't have yet are `food` ~2400 `newspapers` ~2900 `condoms` ~2300

I will add them! Can you tell me the BE, RU and UK translations for those types so I can add them too? Thanks!

File: data/categories.txt (hunk 4 above)
Comment (dvdmrtnz) [reply data/categories.txt:7973]: What do those number 4 or number 3 on the categories mean? I have seen them on other categories but couldn’t figure out what they are for.

**Comment by pastk** [comment]
> 
> I will add them! Can you tell me the BE, RU and UK translations for those types so I can add them too? Thanks!

Then maybe you can also merge `water` (~1200) and `milk` (~1100) into existing `vending=drinks` using `replaced_tags.txt`? And add `bread` (~1000) into `food`?

```
[type.amenity.vending_machine.food]
    be = Аўтамат з ежай
    ru = Автомат с едой
    uk = Автомат з їжею

amenity-vending_machine-food|@food
be:Аўтамат з ежай|5перакус|4снэкі
ru:Автомат с едой|5перекус|4снэки
uk:Автомат з їжею|5перекус|4снекі


[type.amenity.vending_machine.condoms]
    be = Аўтамат з прэзерватывамі|
    ru = Автомат с презервативами
    uk = Автомат з презервативами

amenity-vending_machine-condoms
be:Аўтамат з прэзерватывамі|5прэзерватывы|5кандамат
ru:Автомат с презервативами|5презервативы|5кондомат
uk:Автомат з презервативами|5презервативи|5кондомат

[type.amenity.vending_machine.newspapers]
    be = Газетны аўтамат
    ru = Газетный автомат
    uk = Газетний автомат

amenity-vending_machine-newspapers
be:Газетны аўтамат|газетамат|3газеты|4прэса
ru:Газетный автомат|газетомат|3газеты|4пресса
uk:Газетний автомат|газетомат|3газети|4преса
```

**Review by vng — APPROVED** [review]

File: data/mapcss-mapping.csv (hunk 1 above)
Comment (vng) [reply data/mapcss-mapping.csv]: Well, yes in general, but not always.

File: data/categories.txt (hunk 4 above)
Comment (pastk) [reply data/categories.txt:8007]: Ahh, this is to indicate the number of symbols that need to be typed in a search query to make this keyword appear in the list of search suggestions / hints. Its described in the description in the beginning of `categories.txt` though doesn't pop up to the eyes immediately..

File: data/categories.txt [hunk 6]
```diff
@@ -8023,6 +8039,14 @@ el:Αυτόματος πωλητής ποτών
 sk:Nápojový automat
 fa:ماشین فروش نوشیدنی
 
+amenity-vending_machine-food|@food
+en:Food Dispenser|Food
```
Comment (pastk) [inline data/categories.txt:8043]: \+ en:Snacks

File: data/categories.txt [hunk 7]
```diff
@@ -7989,6 +7989,22 @@ el:Αυτόματος πωλητής τσιγάρων
...
+amenity-vending_machine-coffee
+en:4Coffee Dispenser|Coffee
+be:3Кававы аўтамат|кава
+es:Máquina expendedora de café|Café
+fr:Distributeur de café|Café
+ru:3Кофейный автомат|кофе|кофемат
+uk:3Кавовий автомат|кава
+
+amenity-vending_machine-condoms
+en:Condoms Dispenser|Condoms
```
Comment (pastk) [inline data/categories.txt:8001]: \+ en:Condomat

**Review by pastk — APPROVED** [review]
Looks great, thanks!

Please add auto-suggestion numbers to search keywords if you wish.

**Comment by dvdmrtnz** [comment]
> Looks great, thanks!
> 
> Please add auto-suggestion numbers to search keywords if you wish.

Done

File: data/categories.txt (hunk 4 above)
Comment (dvdmrtnz) [reply data/categories.txt:8007]: I see it now! Thanks!

**Pull request merged**